lib/virt_autotest/utils.pm

```diff
@@ -55,7 +56,7 @@ sub restart_libvirtd {
     else {
```

Comment:
I understand that this is a draft PR only. Just to share the comment after I quickly go through the PR, in case it helps.

To me, the core part of this task/poo ticket, is to replace the monolithic libvirt operations with the equivalent operations in modular libvirt, for whatever products that are using modular libvirt. Typical operations include, but not limited to, libvirtd restart/stop/start/status, enabling debug logs for qemu/libvirt/xen. In other words, whatever libvirtd related operations that are used in our automation in openqa/qa_lib_virtauto, should be provided the new equivalent code in modular libvirt, and make all tests work, in similar test logic, but different real code handling.

Given that it is obvious that in our code, there are multiple calling to such libvirt functions, so to have less code change, and let users of these lib functions not need to worry about modular/monolithic libvirt differences, it will be good to add such different handling for modular libvirt *directly within each lib function*. For example, here in `sub restart_libvirtd`, you can add logic like below:

```
sub restart_libvirtd {
    if  (is_sle('<12')) {}
    elsif (is_alp) {}
    elsif (! is_modular_libvirt) {}
    else {restart_modular_libvirt_daemons} # modular libvirt handling
```

Reply:
Good point. much more make sense now.

Reply:
Short update, update restart_libvirtd() as below: 

```
sub restart_libvirtd {
    if  (is_sle('<12')) {}
    elsif (is_alp) {}
    elsif (is_modular_livirtd) {restart_modular_libvirt_daemons}
    else {restart libvirtd} # monolithic libvirtd  
```